app/controllers/api/v1/registrations_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class RegistrationsController < ApplicationController
+      skip_before_action :doorkeeper_authorize!
+
+      def create
+        validation = RegistrationForm.new(sign_up_params)
```

Comment:
Follow the convention, we will pass the param to the `save(params)` instead of the `initializer` method

Reply:
```suggestion
        registration_form = RegistrationForm.new
```

Also let's name it `regiastration_form` or `form` ;-)